backend/apps/volontulo/urls.py

```diff
@@ -144,8 +145,11 @@
         name='organization_form'
     ),
     # organizations/filter
-    # organizations/<slug>/<id>/contact
-
+    url(
```

Comment:
Two things here:
 1. We started to put `/api` routes at the beginning of `urlpatterns` so it would be great to stick with this convention
 2. I would propose to use action URLs here instaed of separate api view: (see: http://www.django-rest-framework.org/api-guide/viewsets/#viewset-actions)